styles/card.css

```diff
@@ -0,0 +1,153 @@
+.card-container-main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: 1rem;
+}
+
+.card-heading {
+    margin-left: 0.3rem;
+}
+
+.card-author {
+    padding: 2px;
+    margin: 0.3rem;
+}
+
+.card-content {
+    margin: 0.5rem;
+}
+
+.card-container {
+    background-color: var(--bg-body);
+    width: 22rem;
+    border-radius: 1rem;
+    border: 1px solid var(--primary-color);
+}
+
+.card-container-horizontal {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 1rem;
+}
+
+.card-horizontal {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+}  
+
+.card-image {
+    max-inline-size: 100%;
+    block-size: auto;
+    border-radius: 1rem 1rem 0 0;
+    
+}
+
+.image-horizontal {
+    border-radius: 0px;
+    width: 100%;
+}
+
+.card-actions {
+    display: flex;
+    justify-content: space-around;
+    margin: 0.5rem 1rem 1rem 1rem;
+    padding: 0.3rem;
+    cursor: pointer;
+}
+
+.card-btn {
+    text-decoration: none;
+    background: transparent;
+    border-radius: 10px;
+    font-weight: bolder;
+    font-size: 1rem;
+    color: var(--primary-color);
+    background-color: var(--bg-body);
+    cursor: pointer;
+}
+
+.card-icons {
+    display: flex;
+    justify-content: flex-end;
+    gap: 2.5rem;
+    margin-right: 1.5rem;
+    margin-bottom: 1rem;
```

Comment:
You can use shorthand properties here.

Reply:
Yeah. Will update it. Thanks for pointing out.